### PR TITLE
[miio] additional vacuum channels for advanced rules 

### LINKED ISSUE #10144: [miio] add channel for room16 in order to find current floor
For several [rules](https://community.openhab.org/t/cleaning-specific-room-and-zones-with-multi-level-map-roborock/115329/8?u=peterk) it is important that OpenHAB is aware of the floor where the vacuum is cleaning. Unfortunately the vacuum has no possibility to show this through the binding.

In this [thread](https://community.openhab.org/t/xiaomi-roborock-room-clean-multi-level-map/108850/13?u=peterk) is a nice workaround. When sending the command `get_room_mapping`, the result is unique for each floor.

```
//Second Floor Map
//{"id":8830,"result":[[16,"536001005515"],[17,"536001005513"],[18,"536001005511"],[20,"536001005519"],[21,"536001005517"]],"exe_time":100}

//First Floor Map
//{"id":12270,"result":[[16,"536001005545"],[17,"536001005547"],[18,"536001005549"]],"exe_time":100}

```

Using this information, it is possible to use JSONPATH to get the unique ID of Room 16 and then use MAP-Transformation in order to get the floor.

```
Etage = transform("MAP", "Rocky_Etage.map", transform("JSONPATH", "$.result[0][1]",newState.toString))
```

The map-file has the mapping as e.g.

```
536001005515=Second
536001005545=First
=UNDEF

```
if there is no room 16 or there is no mapping, the floor will be set to UNDEF.

This whole set of rules would vanish, if there is a new channel always presenting the ID of room 16. The mapping will then be done using a profile for MAP-transformation.

The resulting Item will always present the current floor without any rules.

### PR DESCRIPTION
**Pull request by marcelrv**
closes #10144

### CODE REVIEW

**Comment by cpmeister** [comment]
Can you update the readme to reflect the new channels?

**Comment by marcelrv** [comment]
> Can you update the readme to reflect the new channels?

Yes, done.

File: bundles/org.openhab.binding.miio/README.md [hunk 1]
```diff
@@ -2663,7 +2663,7 @@ e.g. `openhab:send actionCommand 'upd_timer["1498595904821", "on"]'` would enabl
 |----------------------|----------------------|------------------------------------------|------------|
 | power                | Switch               | Power                                    |            |
 | brightness           | Dimmer               | Brightness                               |            |
-| ambientBrightness    | Number               | Ambient Brightness                       |            |
+| ambientBrightness    | Dimmer               | Ambient Brightness                       |            |
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/README.md:2666]: What does this change have to do with this PR?

Comment (marcelrv) [reply bundles/org.openhab.binding.miio/README.md:2666]: Nothing, 
It seems this was missed in the previous run, so it was picked up in the new run

**Review by cpmeister — APPROVED** [review]

**Pull request merged**